services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -33,7 +33,10 @@
     is_pipeline_running,
     is_pipeline_stopped,
 )
-from ...utils.dags import create_dag_graph, create_minimal_graph_based_on_selection
+from ...utils.dags import (
+    create_complete_dag_graph,
```

Comment:
MINOR: since names are growing ... :-) I would like to suggest some alternativs:

- ``daG`` includes the word graph in it so : ``create_complete_dag`` or ``create_complete_graph``
- these DAGs include only computational nodes, right? so ``create_minimal_dag_from_selection`` would work?

Reply:
no. actually the minimal_computation_dag is a computational dag as its name implies.
the complete dag is the full thing